signingscript/test/test_sign.py

```diff
@@ -84,6 +84,8 @@ def context_die(*args, **kwargs):
         target_path = os.path.join(tmpdir, os.path.relpath(path, BASE_DIR))
         assert os.path.exists(target_path)
         assert os.path.isfile(target_path)
+        assert os.stat(target_path).st_uid == 0
+        assert os.stat(target_path).st_gid == 0
```

Comment:
this won't work unless the test is running as root...The extracted file should have owner/group the same as the current process, whereas the files in the tarball should have 0/0 ownership.